dll/win32/shell32/shelldesktop/CChangeNotify.h

```diff
@@ -6,79 +6,64 @@
  */
 #pragma once
 
-#define INVALID_REG_ID 0
+#define INVALID_REG_ID 0 /* invalid registration ID */
 
-#define WM_GETDELIWORKERWND (WM_USER + 25) /* 0x419 */
-#define WM_OLDDELI_HANDOVER (WM_USER + 1) /* 0x401 */
-#define WM_NOTIF_REG (WM_USER + 1) /* 0x401 */
-#define WM_NOTIF_UNREG (WM_USER + 2) /* 0x402 */
-#define WM_NOTIF_DELIVERY (WM_USER + 3) /* 0x403 */
-#define WM_NOTIF_SUSPEND (WM_USER + 6) /* 0x406 */
-#define WM_NOTIF_REMOVEBYPID (WM_USER + 7) /* 0x407 */
+#define WM_SHELL_GETWORKERWND (WM_USER + 25) /* 0x419 */
```

Comment:
let's call this WM_DESKTOP_GET_CNOTIFY_WORKER

Reply:
OK.